com.vicr123.thebeat.yml

```diff
@@ -0,0 +1,93 @@
+app-id: com.vicr123.thebeat
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: thebeat
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --share=network
+  - --own-name=org.mpris.MediaPlayer2.theBeat
+  - --filesystem=home
```

Comment:
Any reason this is not xdg-music?

Reply:
There's an add to library function which simply tells the app to search for music in a different folder. Are permissions for that folder preserved between each startup?

Reply:
To be honest, I'm not certain. If I execute `flatpak permission-list | grep -E 'read|write`, I see a number of files that I would assume application still have access to, but I don't know how uniform directory support across distros is. Let's keep it as is.